src/mapgen/mg_ore.cpp

```diff
@@ -170,7 +170,7 @@ void OreScatter::generate(MMVManip *vm, int mapseed, u32 blockseed,
 
 		if (biomemap && !biomes.empty()) {
 			u32 index = sizex * (z0 - nmin.Z) + (x0 - nmin.X);
-			std::unordered_set<u8>::const_iterator it = biomes.find(biomemap[index]);
+			std::unordered_set<u16>::const_iterator it = biomes.find(biomemap[index]);
```

Comment:
It would make more sense to just change the type to `auto`

Reply:
I cannot judge that as i do not understand how auto works or how to apply it.
Do you mean:
`std::unordered_set<auto>::const_iterator it = biomes.find(biomemap[index]);`
?
I will need specific instructions if you want me to do this. Alternatively, this could be done in a follow-up PR by a core dev who understands what they are doing.

Reply:
When you have a complicated looking type returned by `.find()` or similar methods it's a good idea to replace the entire type of the variable to `auto`.
In this case: `auto it = biomes.find(biomemap[index]);`

Reply:
Thanks, will do this.